File: src/main/java/akko/ddbot/data/BilibiliApi/BilibiliDataClass.java
package akko.ddbot.data.BilibiliApi;

public class BilibiliDataClass {

    private int code;
    private String message;
    private int ttl;
    private Data data;

    public int getCode() {
        return code;
    }

    public String getMessage() {
        return message;
    }

    public int getTtl() {
        return ttl;
    }

    public Data getData() {
        return data;
    }

}

class Official {

    private int role;
    private String title;
    private String desc;
    private int type;

    public int getRole() {
        return role;
    }

    public String getTitle() {
        return title;
    }

    public String getDesc() {
        return desc;
    }

    public int getType() {
        return type;
    }

}

class Vip {

    private int type;
    private int status;
    private int theme_type;
    private Label label;
    private int avatar_subscript;
    private String nickname_color;

    public int getType() {
        return type;
    }

    public int getStatus() {
        return status;
    }

    public int getTheme_type() {
        return theme_type;
    }

    public Label getLabel() {
        return label;
    }

    public int getAvatar_subscript() {
        return avatar_subscript;
    }

    public String getNickname_color() {
        return nickname_color;
    }

}

class Label {

    private String path;
    private String text;
    private String label_theme;

    public String getPath() {
        return path;
    }

    public String getText() {
        return text;
    }

    public String getLabel_theme() {
        return label_theme;
    }

}

class Pendant {

    private int pid;
    private String name;
    private String image;
    private int expire;
    private String image_enhance;

    public int getPid() {
        return pid;
    }

    public String getName() {
        return name;
    }

    public String getImage() {
        return image;
    }

    public int getExpire() {
        return expire;
    }

    public String getImage_enhance() {
        return image_enhance;
    }

}

class Nameplate {

    private int nid;
    private String name;
    private String image;
    private String image_small;
    private String level;
    private String condition;

    public int getNid() {
        return nid;
    }

    public String getName() {
        return name;
    }

    public String getImage() {
        return image;
    }

    public String getImage_small() {
        return image_small;
    }

    public String getLevel() {
        return level;
    }

    public String getCondition() {
        return condition;
    }

}

class Theme {

}

class Sys_notice {

}

File: src/main/java/akko/ddbot/data/BilibiliApi/BilibiliDataClass.kt
package akko.ddbot.data.BilibiliApi

data class BilibiliDataClass(
        val code: Int,
        val `data`: Data,
        val message: String,
        val ttl: Int
)

data class Data(
        val birthday: String,
        val coins: Int,
        val face: String,
        val fans_badge: Boolean,
        val is_followed: Boolean,
        val jointime: Int,
        val level: Int,
        val live_room: LiveRoom,
        val mid: Int,
        val moral: Int,
        val name: String,
        val nameplate: Nameplate,
        val official: Official,
        val pendant: Pendant,
        val rank: Int,
        val sex: String,
        val sign: String,
        val silence: Int,
        val sys_notice: SysNotice,
        val theme: Theme,
        val top_photo: String,
        val vip: Vip
)

data class LiveRoom(
        val broadcast_type: Int,
        val cover: String,
        val liveStatus: Int,
        val online: Int,
        val roomStatus: Int,
        val roomid: Int,
        val roundStatus: Int,
        val title: String,
        val url: String
)

data class Nameplate(
        val condition: String,
        val image: String,
        val image_small: String,
        val level: String,
        val name: String,
        val nid: Int
)

data class Official(
        val desc: String,
        val role: Int,
        val title: String,
        val type: Int
)

data class Pendant(
        val expire: Int,
        val image: String,
        val image_enhance: String,
        val name: String,
        val pid: Int,
        val image_enhance_frame: String
)

class SysNotice

class Theme

data class Vip(
        val avatar_subscript: Int,
        val label: Label,
        val nickname_color: String,
        val status: Int,
        val theme_type: Int,
        val type: Int
)

data class Label(
        val label_theme: String,
        val path: String,
        val text: String
)